ftests/pom.xml

```diff
@@ -13,37 +13,60 @@
 
   <properties>
     <maven.deploy.skip>true</maven.deploy.skip>
+    <nuxeo.ftests.tier5>false</nuxeo.ftests.tier5>
+    <nuxeo.ftests.tier6>false</nuxeo.ftests.tier6>
   </properties>
 
-  <modules>
-    <module>nuxeo-3d-jsf-ui-ftests</module>
-    <!-- NXP-28036: ignored -->
-    <!-- <module>nuxeo-amazon-s3-jsf-ui-ftests</module> -->
-    <module>nuxeo-csv-jsf-ui-ftests</module>
-    <module>nuxeo-diff-jsf-ui-ftests</module>
-    <module>nuxeo-diff-pictures-jsf-ui-ftests</module>
-    <module>nuxeo-drive-jsf-ui-ftests</module>
-    <module>nuxeo-easyshare-jsf-ui-ftests</module>
-    <module>nuxeo-groups-rights-audit-jsf-ui-ftests</module>
-    <module>nuxeo-imap-connector-jsf-ui-ftests</module>
-    <module>nuxeo-jsf-ui-hotreload-tests</module>
-    <module>nuxeo-jsf-ui-webdriver-tests</module>
-    <module>nuxeo-lang-ext-incomplete-jsf-ui-ftests</module>
-    <module>nuxeo-liveconnect-jsf-ui-ftests</module>
-    <!-- NXP-28036: ignored -->
-    <!-- <module>nuxeo-microsoft-azure-jsf-ui-ftests</module> -->
-    <module>nuxeo-multi-tenant-jsf-ui-ftests</module>
-    <module>nuxeo-platform-error-web-tests</module>
-    <module>nuxeo-platform-forms-layout-demo-tests</module>
-    <module>nuxeo-quota-jsf-ui-ftests</module>
-    <module>nuxeo-shibboleth-invitation-jsf-ui-ftests</module>
-    <module>nuxeo-shibboleth-login-jsf-ui-ftests</module>
-    <module>nuxeo-signature-jsf-ui-ftests</module>
-    <module>nuxeo-smart-search-jsf-ui-ftests</module>
-    <module>nuxeo-template-rendering-jsf-ui-ftests</module>
-    <module>nuxeo-tree-snapshot-jsf-ui-ftests</module>
-    <module>nuxeo-virtual-navigation-jsf-ui-ftests</module>
-  </modules>
+  <profiles>
+    <profile>
+      <id>ftestsTier5</id>
+      <activation>
+        <property>
+          <name>nuxeo.ftests.tier5</name>
+        </property>
+      </activation>
+      <modules>
+        <module>nuxeo-jsf-ui-hotreload-tests</module>
```

Comment:
I would still prefer hot reload tests to come after standard tests: what prompted me to want to put "most relevant" tests first is linked to the fact that I thought I broke themes hot reload while I simply broke themes (and keeping alphabetical order does not seem that important here?)